library/func_misc.py

```diff
@@ -206,7 +207,9 @@ def Call(self, args):
 
         strs = []  # type: List[str]
         for i, el in enumerate(li):
-            strs.append(val_ops.Stringify(el, loc.Missing))
+            strs.append(
+                val_ops.Stringify(
+                    el, location.TokenForListElement(args.PosNode(0), i)))
```

Comment:
This is a good example of non-trivial use (from `join()`)

Reply:
Need to make this, and the other `val_ops` calls throughout less eager...

Reply:
Actually, just this one. The others are just calling `Reader::BlamePos()`, which is cheap